docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
             - newsroom
 
     mongo:
-        image: mongo:3.6
+        image: mongo:6
```

Comment:
Would be good to also update elastic to at least `7.17.22`, for this and e2e/docker-compose (that's what our test infra now uses). https://github.com/superdesk/superdesk-core/blob/async/docker-compose.yml#L9